api/pkg/transformer/feast/bigtablestore/storage_test.go

```diff
@@ -0,0 +1,59 @@
+package bigtablestore
```

Comment:
How did you generate this file? Is it possible to add that information in this file too? Similar to what we've done with mockery.

Reply:
it is basically generated by mockery but I moved to the same package, because the interface is not exposed publicly